src/transitions/scroll-transition.js

```diff
@@ -171,10 +173,21 @@ export default class ScrollTransition extends React.Component {
             MIN_ZOOM_SCALE
           )
         : 1.0;
+    const listWidth =
+      this.props.slideWidth * React.Children.count(this.props.children);
+    const spacingOffset =
+      this.props.cellSpacing * React.Children.count(this.props.children);
+    const calculatedHeight = this.props.vertical
+      ? listWidth + spacingOffset
+      : this.props.slideHeight;
+
     return {
       boxSizing: 'border-box',
       display: this.props.vertical ? 'block' : 'inline-block',
-      height: 'auto',
+      height:
+        calculatedHeight > 0 && this.props.heightMode !== 'current'
+          ? calculatedHeight
+          : 'auto',
```

Comment:
When the carousel first loads the calculatedHeight will be 0, so we need to let it render with `auto` at first so the heights can be correctly registered and then we swap out the `auto` to a fixed height.